forge/db/views/Project.js

```diff
@@ -91,14 +91,38 @@ module.exports = {
             return result
         }))
     },
+    instancesSummaryList: function (app, instancesArray) {
+        return instancesArray.map((instance) => {
+            // Full settings are not
```

Comment:
[nb] Not sure what this comment is trying to say!

Reply:
Comment removed